packages/compiler/src/render3/view/template.ts

```diff
@@ -1017,21 +1020,15 @@ export class TemplateDefinitionBuilder implements t.Visitor<void>, LocalResolver
    *   STYLES, style1, value1, style2, value2,
    *   SELECT_ONLY, name1, name2, name2, ...]
    * ```
+   *
+   * Note that this function will fully ignore all synethic (@foo) attribute values
```

Comment:
```suggestion
   * Note that this function will fully ignore all synthetic (@foo) attribute values
```

Given the name change, I'm also wondering if this top-level comment is relevant still?